lib/internal/Magento/Framework/Module/ModuleList/Sorter.php

```diff
@@ -0,0 +1,83 @@
+<?php
+/**
+ * @copyright Copyright (c) 1999-2017 netz98 GmbH (http://www.netz98.de)
```

Comment:
I assume this is a php storm autocomplete. I guess this should be the Magento docblock instead.

Reply:
:-) Yes, you catched me. I will change the PhpStorm template.